File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/RevocationRegistryFullException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when attempting to use a full revocation registry.
    /// </summary>
    public class RevocationRegistryFullException : IndyException
    {
        const string message = "The specified revocation registry is full.  Another revocation registry must be created.";

        /// <summary>
        /// Initializes a new RevocationRegistryFullException.
        /// </summary>
        internal RevocationRegistryFullException() : base(message, (int)ErrorCode.AnoncredsRevocationRegistryFullError)
        {

        }
    }

}

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/anoncreds/RevocationRegistryFullException.java
package org.hyperledger.indy.sdk.anoncreds;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when a revocation registry is full.
 */
public class RevocationRegistryFullException extends IndyException
{
	private static final long serialVersionUID = 8294079007838985455L;
	private final static String message = "The specified revocation registry is full.  Another revocation registry must be created.";

	/**
	 * Initializes a new RevocationRegistryFullException.
	 */
	public RevocationRegistryFullException()
	{
		super(message, ErrorCode.AnoncredsRevocationRegistryFullError.value());
	}
}